src/store/user-store.ts

```diff
@@ -16,16 +19,46 @@ class UserStore {
     return !!this.user?.id;
   }
 
+  public init = async () => {
+    this.isInitLoading = true;
+    this.error = '';
+
+    try {
+      const userID = localStorage.getItem(LSKeys.USER_ID);
+      if (userID) {
+        const response = await customerService.getCustomerByID(userID);
+        runInAction(() => {
+          console.log(response);
+          this.user = response;
```

Comment:
тоже, не супер важно, но раз мы учимся, то чеб и нет. называть переменную response есть смысл если туда приходят разные поля, типа data, error. в твоем случае туда приходит сразу кастомер. и метод у тебя называется гетКастомер. соответственно более читаемо было бы
```suggestion
        const customer = await customerService.getCustomerByID(userID);
        runInAction(() => {
          console.log(customer);
          this.user = customer;
```